snowflake-uar-reviews/README.md

```diff
@@ -0,0 +1,70 @@
+# User Access Reviews by Snowflake
+
+## Why you need this
+This Retool app makes it easy to perform periodic reviews of employee access rights. Navigate the process of preparing data, reviewing access rights, and remediating within an easy-to-use GUI. 
+
+
+
+## What does the User Access Review app do?
+Provides an interface to re-evaluate user roles, access privileges, and credentials to mitigate insider threats, prevent insider mistakes, reduce excessive access, and meet compliance requirements such as HIPAA, FedRAMP, SOX, and more. Take actions such as approving or revoking privileges and enter your reasons for doing so. Submit individual changes or update multiple records in bulk. 
+
+This repo includes all code needed to source user roles within your Snowflake instance and generate the table schemas that the application attaches to. Once you load this application, you have the ability to modify the source code as you need to align to your own organization’s processes. This repo: 
+
+* Provides a schema to use on Snowflake
+* Provides a manager review application that implements filtering at the manager level
+* An admin application to get started with user and role entry
+* A review application to audit manager submissions for super-users and compliance teams
```

Comment:
To get the case of all of the items in this list to agree, I'd start the list with "This repo provides:" and then remove " Provides" from the start of the first two bullets.